rubrics/api/src/main/java/org/sakaiproject/rubrics/api/model/Rating.java

```diff
@@ -70,6 +72,8 @@ public class Rating implements PersistableEntity<Long>, Serializable {
     @Column(nullable = false)
     private Double points;
 
+    private Double weightedPoints;
```

Comment:
I think you mean this to be transient as it is factored by a calculation. You should use the Transient annotation, probably, unless you really want a column creating.